### PR TITLE
Add general teleport config to the teleport-bootstrap module

### PR DESCRIPTION
**Pull request by MattiasGees**
We want to add the systemd service file and general config file of teleport to the module so that we can use it in other modules.

We can use this in our vault and kubernetes setup.

Example outputs
```
Outputs:

config = - content: |
    ssh_service:
      enabled: yes
      listen_addr: 0.0.0.0:3022

      commands:
        - name: arch
          command: [uname, -p]
          period: 1h0m0s
      permit_user_env: false
    auth_service:
      enabled: no
    proxy_service:
      enabled: no

  path: /etc/teleport.yaml
```
```
service = - content: |
    [Unit]
    Description=Teleport SSH Service
    After=network.target

    [Service]
    EnvironmentFile=/etc/teleport
    Type=simple
    Restart=on-failure
    ExecStart=/usr/local/bin/teleport start --config=/etc/teleport.yaml --nodename $NODENAME --advertise-ip $ADVERTISE_IP --auth-server $AUTH_SERVER --token $AUTH_TOKEN

    [Install]
    WantedBy=multi-user.target

  path: /lib/systemd/system/teleport.service
```

### CODE REVIEW

**Comment by SamClinckspoor** [comment]
the `--config=/etc/teleport.yaml` overrides the inline params?

File: README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ This module creates a script to configure and start the Teleport service on a se
 
 ### Output
  * [`teleport_bootstrap_script`]: String: The rendered script to add to the Instance cloud-init user data.
+ * [`teleport_config_cloudinit`]: String: The rendered Teleport config that you can add to the instance cloud-init user data
+ * [`teleport_service_cloudinit`]: String: The rendered Teleport systemd service that you can add to the instance cloud-init user data
```
Comment (SamClinckspoor) [inline README.md:61]: to be sure, you need to put these in a `write_files:` block right?

Comment (MattiasGees) [reply README.md:61]: yes

**Comment by MattiasGees** [comment]
@SamClinckspoor no it doesn't. I tested this and it works together.

**Review by SamClinckspoor — APPROVED** [review]

**Review by simonrondelez — APPROVED** [review]

**Pull request merged**